rust/examples-frontend/svc/index.html

```diff
@@ -0,0 +1,50 @@
+<!doctype html>
+<style>
+    body {
+        margin: 0;
+    }
+
+    #container {
+        display: flex;
+    }
+
+    figure {
+        margin: 0;
+        width: 50vw;
+    }
+
+    video {
+        max-width: 100%;
+    }
+
+    #container {
+        display: flex;
+        flex-direction: column;
+        justify-content: space-between;
+        height: 100vh;
+    }
+
+    .previews {
+        display: flex;
+    }
+</style>
+<div id="container">
+    <div class="previews">
+        <figure>
+            <video id="preview-send" muted controls></video>
+            <figcaption>Send preview</figcaption>
+        </figure>
+        <figure>
+            <video id="preview-receive" muted controls></video>
+            <figcaption>Receive preview</figcaption>
+        </figure>
+    </div>
+
+    <div class="controls">
+        <button id="decSL" type="button">down</button>
+        [<span id="SL">none</span>]
+        <button id="incSL" type="button">up</button>
+    </div>
```

Comment:
If this is spatial layer, would be nice to have controls over temporal layers too. Or have it like in mediasoup demo where buttons control both.

Reply:
can you send me a link to this source? cause from what I see they do exactly the same

Reply:
I don't know where it is in the code, try to find it yourself. In mediasoup demo there are controls both on sender and receiver side, you are probably talking about sender side, which is sometimes useful, but not the point of this example because you can do it without simulcast or SVC on sender side anyway.